### ISSUE TITLE
Host this app for free on Heroku or Firebase and provide url

### ISSUE THREAD

**Issue by madnight** [aocole/RosettaCodeExplorer#1]

**Comment by aocole** [comment]
Hi @madnight. Can you tell me more about your use case? This project hasn't been updated in a long time and I would want to make sure the libs are all up-to-date before hosting it anywhere public. If you can tell me about how you would like to use the project maybe we can come up with ideas for improvement at the same time.

**Comment by madnight** [comment]
Hi @aocole i would like to have an easier public hosted interface access to the code samples from Rosetta Code. Maybe also adding a simple API.

**Comment by aocole** [comment]
I don't think this is something that I'm going to take on. The code is open and you are welcome to create a public hosted version if you like. Thanks for your interest in the project!
